Tasks/KubernetesManifestV0/src/utils/utilities.ts

```diff
@@ -147,7 +148,21 @@ export function substituteImageNameInSpecFile(currentString: string, imageName:
     }, '');
 }
 
+export function getAPIVersion() {
+    if(apiVersion)
+        return apiVersion;
+
+    const kubectlPath = tl.which('kubectl', true)
+    const command = tl.tool(kubectlPath);
+    command.arg('api-versions');
+    const result = command.execSync().stdout.split('\n');
+    apiVersion = result.find(version => version.startsWith('split.smi-spec.io'));
+
+    return apiVersion = (apiVersion != null && typeof apiVersion != 'undefined') ? apiVersion : 'split.smi-spec.io/v1alpha1';
```

Comment:
If it was unable wouldn't the deployment fail?

Reply:
Throw a warning otherwise

Reply:
I'll add that